scripts/lib/mkcloud-onhost.sh

```diff
@@ -148,7 +148,7 @@ function onhost_cacheclouddata
                 echo "images/$a/other/magnum-service-image.qcow2"
                 echo "images/$a/other/manila-service-image.qcow2"
                 # these are the real ones, the above are just symlinks
-                echo "images/$a/other/fedora-23-atomic-7.qcow2"
+                echo "images/$a/other/Fedora-Atomic-26.x86_64.qcow2"
```

Comment:
Isn't `$a` already the architecture? then please remove it from the image name.

Reply:
That's something @dirkmueller will need to change first if needed.

Reply:
Changed the img name.